openy_location/modules/openy_loc_branch/config/optional/field.field.node.branch.field_location_address.yml

```diff
@@ -0,0 +1,52 @@
+langcode: en
```

Comment:
this config should go to original place https://github.com/open-y-subprojects/openy_features/blob/main/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_address.yml